Cargo.toml

```diff
@@ -17,3 +17,7 @@ png = "0.15.3"
 
 [dependencies.allsorts]
 version = "0.5.0"
+
+[dev-dependencies]
+assert_cmd = "0.10"
```

Comment:
Any reason why you aren't using v1 of `assert_cmd`?

Reply:
Not a good one! I copied that section from https://rust-cli.github.io/book/tutorial/testing.html (will fix)

Reply:
yeah just noticed when I copy pasted this into my editor - my crates.io thingy said there was a new version out 😅